### PR TITLE
GraphQL: added Contract inserted_at, updated_at fields

### PR DESCRIPTION
**Pull request by PavelVesnin**

### CODE REVIEW

File: apps/graphql/test/web/resolvers/contract_resolver_test.exs [hunk 1]
```diff
@@ -445,6 +445,8 @@ defmodule GraphQLWeb.ContractResolverTest do
             contractRequest {
               databaseId
             }
+            inserted_at
```
Comment (progsmile) [inline apps/graphql/test/web/resolvers/contract_resolver_test.exs]: shouldn't be `insertedAt` ?

File: apps/graphql/test/web/resolvers/contract_resolver_test.exs [hunk 2]
```diff
@@ -471,7 +473,11 @@ defmodule GraphQLWeb.ContractResolverTest do
 
       resp_entity = get_in(resp_body, ~w(data contract))
 
-      assert nil == resp_body["errors"]
+      refute resp_body["errors"]
+
+      assert resp_entity["inserted_at"]
```
Comment (progsmile) [inline apps/graphql/test/web/resolvers/contract_resolver_test.exs]: Does that assertions work? `insertedAt` should be also in response

Comment (kiurchv) [inline apps/graphql/test/web/resolvers/contract_resolver_test.exs]:
```suggestion
      assert resp_entity["insertedAt"]
```

File: apps/graphql/test/web/resolvers/contract_resolver_test.exs [hunk 3]
```diff
@@ -471,7 +473,11 @@ defmodule GraphQLWeb.ContractResolverTest do
...
+      assert resp_entity["updated_at"]
```
Comment (kiurchv) [inline apps/graphql/test/web/resolvers/contract_resolver_test.exs]:
```suggestion
      assert resp_entity["updatedAt"]
```

File: apps/graphql/test/web/resolvers/contract_resolver_test.exs (hunk 1 above)
Comment (PavelVesnin) [reply apps/graphql/test/web/resolvers/contract_resolver_test.exs]: should

File: apps/graphql/test/web/resolvers/contract_resolver_test.exs (hunk 2 above)
Comment (PavelVesnin) [reply apps/graphql/test/web/resolvers/contract_resolver_test.exs]: works :)

**Pull request merged**